documentation/src/main/asciidoc/topics/proc_server_configuring_ssl_engine.adoc

```diff
@@ -1,21 +1,22 @@
-[id='ssl_engine-{context}']
-= Tuning SSL Protocols and Cipher Suites
-You can configure the SSL engine, via the {brandname} server SSL identity, to
-use specific protocols and ciphers.
-
-[IMPORTANT]
-====
-You must ensure that you set the correct ciphers for the protocol features you
-want to use; for example HTTP/2 ALPN.
-====
+[id='configuring-ssl-engine_{context}']
+= Configuring TLS versions and cipher suites
+When using SSL/TLS encryption to secure your deployment, you should configure {brandname} Server to use specific versions of the TLS protocol as well as specific cipher suites within the protocol.
```

Comment:
why "should configure" protocol and ciphers? Aren't the defaults OK to use?

Reply:
thanks @pruivo I've updated s/should/can. I also added a note about omitting the `enabled-protocols` attribute, which allows any TLS version based on a comment from Wolf.